collab-document/src/importer/md_importer.rs

```diff
@@ -0,0 +1,509 @@
+use crate::blocks::{Block, DocumentData, DocumentMeta};
+use crate::document_data::generate_id;
+use markdown::mdast::AlignKind;
+use markdown::{mdast, message, to_mdast, Constructs, ParseOptions};
+use serde_json::{json, Value};
+use std::collections::HashMap;
+
+pub struct MDImporter {}
+
+impl Default for MDImporter {
+  fn default() -> Self {
+    Self::new()
+  }
+}
```

Comment:
```suggestion
#[derive(Default)]
pub struct MDImporter {}
```

Reply:
Maybe you can try: pub struct MDImporter;